src/components/HomePage/HomePage.js

```diff
@@ -0,0 +1,9 @@
+import Card from '../Card'
+import './HomePage.css'
```

Comment:
There is no styling in your `HomePage.css`, I think that is why your tests are not running through. Either add some styles, or delete the file